src/main/java/org/shanerx/tradeshop/itrade/IShopCreateEventListener.java

```diff
@@ -65,14 +62,7 @@ public void onSignChange(SignChangeEvent event) throws InterruptedException {
 			player.sendMessage(colorize(getPrefix() + plugin.getMessages().getString("no-ts-create-permission")));
 			return;
 		}
-		if (!plugin.getAllowedInventories().contains(chest.getType())) {
-			event.setLine(0, ChatColor.DARK_RED + "[iTrade]");
-			event.setLine(1, "");
-			event.setLine(2, "");
-			event.setLine(3, "");
-			player.sendMessage(colorize(getPrefix() + plugin.getMessages().getString("no-chest")));
-			return;
-		}
+
```

Comment:
The iTrade shops dont check for the chest anymore, so that you can have admin shops without any chests as they arn't needed(unless the trade is for potions or enchanted books and similar things)